providers/amazon/tests/system/amazon/aws/example_eks_with_nodegroups.py

```diff
@@ -163,6 +164,9 @@ def delete_launch_template(template_name: str):
         task_id="delete_nodegroup",
         cluster_name=cluster_name,
         nodegroup_name=nodegroup_name,
+        retries=4,
+        retry_delay=duration(seconds=60),
```

Comment:
Why did you go with 60 on this one but 30 on the others?

Given that the backoff is exponential I think a relatively short first attempt is good.

Reply:
That was an accident. I already fixed that and changed all of them to 60 but I can change them to 30 if you'd like

Reply:
Changed to 30 seconds